src/elements/context-menu/context-menu-item.svelte

```diff
@@ -0,0 +1,48 @@
+<svelte:options immutable tag="v-context-menu-item" />
+
+<script lang="ts">
+import cx from 'classnames';
+import { addStyles } from '../../lib/index';
+import { dispatcher } from '../../lib/dispatch';
+
+const dispatch = dispatcher();
+
+addStyles();
+
+export let icon: string;
+export let variant: 'primary' | 'danger' = 'primary';
+export let label: string;
+
+const handleSelect = () => {
+  dispatch('select', { value: label });
+};
+</script>
+
+<div>
+  <button
+    role="menuitem"
+    aria-labelledby={label}
+    class="flex flex-row px-2 py-1.5 hover:bg-light w-full text-left"
+    on:click={handleSelect}
+  >
+    {#if icon}
+      <v-icon
+        name={icon}
+        class={cx('p-1', {
+          'text-gray-6': variant === 'primary',
+          'text-danger-dark': variant === 'danger',
+        })}
+      />
+    {/if}
+
+    <p
+      id={label}
+      class={cx('font-normal', {
+        'text-default': variant === 'primary',
+        'text-danger-dark': variant === 'danger',
+      })}
+    >
+      {label}
```

Comment:
ah yea, i like the idea of using IDs and children for content. even when i was leveraging `label` for the `onselect` behavior here, it felt a bit wrong because your display content might be unwieldy and not great for an event id.

i'm going to leave `label` here for now, but agree we should have this conversation and update components as we move to a more atomic design. is there somewhere, maybe in jira, where we can add a reminder?